README.md

```diff
@@ -1,5 +1,10 @@
 # fuzzywuzzy-rs [![docs.rs badge](https://docs.rs/fuzzywuzzy/badge.svg)](https://docs.rs/fuzzywuzzy) [![crates.io badge](https://img.shields.io/crates/v/fuzzywuzzy.svg)](https://crates.io/crates/fuzzywuzzy)
-Fuzzy string matching like a boss. It uses Levenshtein Distance to calculate the differences between sequences in a simple-to-use package.
+> Fuzzy string matching like a boss. It uses Levenshtein Distance to calculate the differences between sequences in a simple-to-use package.
+> [fuzzywuzzy](https://github.com/seatgeek/fuzzywuzzy)
+
+This is a Rust clone of the python package fuzzywuzzy. We attempt to be 100% compatible with the python version.
+
+At the time of writing, our matching algorithm is based on the difflib implementation results which may, in rare cases, [have slightly different results](https://github.com/seatgeek/fuzzywuzzy/issues/128) compared to the python Levenshtein implementation.
```

Comment:
```suggestion
However, our matching algorithm is based on [difflib](https://github.com/DimaKudosh/difflib/wiki), rather than original's Levenshtein-based implementation. This may, in rare cases, cause the ported functions to [behave differently than their original.](https://github.com/seatgeek/fuzzywuzzy/issues/128) for the ported methods.
```

Reply:
It's based on the python version, not the rust version.